atomic_reactor/utils/retries.py

```diff
@@ -109,6 +111,16 @@ def run_cmd(cmd: List[str]) -> bytes:
             e.stdout.decode(),
             e.stderr.decode(),
         )
+        if cleanup_cmd:
+            try:
+                subprocess.run(cleanup_cmd, check=True, capture_output=True)
+            except subprocess.CalledProcessError as c_e:
+                logger.warning(
+                    "Cleanup command: %s failed:\nSTDOUT:\n%s\nSTDERR:\n%s",
+                    cleanup_cmd[0],
```

Comment:
We might want to log the full cleanup command here

Or log it before running it, like we do for the main command

Reply:
added logging